packages/components/src/TooltipTrigger.test.ts

```diff
@@ -0,0 +1,180 @@
+import { act, screen, within } from '@testing-library/react';
+import userEvent from '@testing-library/user-event';
+import { button, h } from 'react-hyperscript-helpers';
+
+import { icon } from './icon';
+import { getPopupRoot } from './internal/PopupPortal';
+import { renderWithTheme } from './internal/test-utils';
+import { TooltipTrigger, TooltipTriggerProps } from './TooltipTrigger';
+
+describe('TooltipTrigger', () => {
+  const tooltipContent = 'This is a tooltip';
+
+  const renderTooltipTrigger = (
+    props: Partial<TooltipTriggerProps> = {},
```

Comment:
Partial can be dangerious if a default object isn't the baseline of a merge....

Reply:
A default object is the baseline of a merge here: `{ content: tooltipContent }` is the base.